jumanji/environments/combinatorial/binpack/env.py

```diff
@@ -103,20 +105,21 @@ class BinPack(Environment[State]):
 
     def __init__(
         self,
-        instance_generator: InstanceGenerator = None,
+        instance_generator: str = "toy",
         obs_num_ems: int = 60,
         reward_fn: RewardFn = sparse_linear_reward,
         normalize_dimensions: bool = True,
         debug: bool = False,
+        **instance_generator_kwargs: Any,
     ):
         """Instantiate a BinPack environment.
 
         Args:
-            instance_generator: InstanceGenerator responsible for resetting the environment. E.g.
-                can be a random generator to learn generalisation or one that outputs the same
-                instance to do active search on that instance. It must inherit from the
-                InstanceGenerator abstract class. Default to ToyInstanceGenerator that always
-                resets to the same instance with 20 items.
+            instance_generator: string representing the InstanceGenerator responsible for resetting
```

Comment:
can we rename this to something like "instance_generator_type" ? as it is no longer the instance generator but a flag

Reply:
done.